### ISSUE TITLE
Printing PDF on default PDF viewer in a chrome extension's side panel closes all chrome windows

### ISSUE THREAD

**Issue by ninbit** [GoogleChrome/chrome-extensions-samples#1201]
**Describe the bug**
In a Chrome extension using a side panel, clicking the print button of the default chrome PDF viewer within the panel **causes all Chrome windows to close** (possibly all Chrome processes as well). This issue does not occur in a popup or a new tab. Furthermore, no print preview is displayed, and Chrome closes abruptly after about one second.

**To Reproduce**
Steps to reproduce the behavior, or file the issue is found in:

1. Create a sample chrome extension with side panel
2. Embed a PDF file in an iframe
3. Open extension in side panel
4. Click the print button on a default PDF viewer

**Expected behavior**
A print preview windows should be displayed

**Screenshots**
![image](https://github.com/GoogleChrome/chrome-extensions-samples/assets/36826102/baf5040d-94da-4221-8a25-09a4844b7e61)

**Notes**
manifest.json:
```
{
  "manifest_version": 3,
  "name": "PDF Print Crash on Side Panel",
  "version": "1.0",
  "description": "PDF Print Crash on Side Panel",
  "side_panel": {
    "default_path": "sidePanel.html"
  },
  "permissions": [
    "sidePanel",
    "tabs"
  ],
  "background": {
    "service_worker": "service_worker.js"
  },
  "action": {
    "default_popup": "sidePanel.html"
  }
}
```

sidePanel.html:
```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>PDF Print Crash on Side Panel</title> 
</head>
<body>           
    <iframe src="doc.pdf" style="width:100%;height: 800px;"></iframe>
</body>
</html>
```

service_worker.js:
`chrome.sidePanel.setPanelBehavior({ openPanelOnActionClick: true });`

You can include a sample doc.pdf file with above files or download all extension files from [here](https://drive.google.com/file/d/1Q68mIeod0HS_62m9B4PeBMsWTfvpsJdT/view?usp=sharing)

I don't know whether it is a bug or I'm missing something in manifest or something else. Any help will be appreciated.

**Comment by oliverdunk** [comment]
Hi, thanks for the report! I was able to reproduce easily using the steps you described.

We don't track Chromium bugs in this repository, so I've filed an associated report [here](https://issues.chromium.org/349554812). Unfortunately, crash bugs are limited visibility by default. I'm looking into if we can open it up but in the meantime I will post here if there are any updates.

Closing this out in favour of that bug.
